src/Http/Http/test/StreamPipeReaderTests.cs

```diff
@@ -565,8 +634,7 @@ public override async Task<int> ReadAsync(byte[] buffer, int offset, int count,
                 await Task.Yield();
                 return await base.ReadAsync(buffer, offset, count, cancellationToken);
             }
-
-#if NETCOREAPP2_2
+#if NETCOREAPP3_0
```

Comment:
Since this is at least the 4th time it's come up, add a comment that we're keeping these due to the planned move to Pipelines.

Reply:
It may be better to just remove them and readd them once we port to Corefx. There are only ~5 directives total and I think I accidentally removed some too.

Reply:
Nit: Newline

Also, the directives StreamPipeWriterTests could use a comment or removal.